backend/src/bundles/auth/auth.service.ts

```diff
@@ -37,14 +39,33 @@ class AuthService {
         const isVerifyUser = await this.verifySignUpCredentials(userRequestDto);
         if (isVerifyUser) {
             const newUser = await this.userService.create(userRequestDto);
-            const { id } = newUser.toObject();
+            const { id, email } = newUser.toObject();
             const token = this.tokenService.createToken({ id });
+            void this.sendAfterSignUpEmail(email);
             return {
                 token,
             };
         }
     }
 
+    private async sendAfterSignUpEmail(email: string): Promise<void> {
+        const htmlToSend = getTemplate({
+            name: 'sign-up-email-template',
+            context: {
+                title: 'SmartSpend',
+                dashboardLink: 'https://smartspend.netlify.app/dashboard',
```

Comment:
I think it's better to move `https://smartspend.netlify.app`  to .env

Reply:
Resolved in [89e9e0b](https://github.com/BinaryStudioAcademy/bsa-winter-2022-2023-smartspend/pull/283/commits/89e9e0b34988b69c1d7370414a32155ea867869a)